test/test_server.c

```diff
@@ -212,7 +220,7 @@ static void set_namespace(int local_size, int univ_size,
     }
     PMIx_generate_ppn(ranks, &ppn);
     free(ranks);
-    PMIX_INFO_LOAD(&info[5], PMIX_PROC_MAP, ppn, PMIX_REGEX);
+    PMIX_INFO_LOAD(&info[5], PMIX_PROC_MAP, ppn, PMIX_STRING);
```

Comment:
I believe this needs to be ```PMIX_REGEX``` as ```ppn``` was generated by ```PMIx_generate_ppn``` and may be a binary blob instead of a string.

Reply:
Fixed with separate commit.